fields.go

```diff
@@ -134,21 +134,17 @@ func (mf *mysqlField) scanType() reflect.Type {
 		fieldTypeMediumBLOB, fieldTypeLongBLOB, fieldTypeBLOB,
 		fieldTypeVarString, fieldTypeString, fieldTypeGeometry, fieldTypeJSON,
 		fieldTypeTime:
-		if mf.flags&flagNotNULL != 0 {
-			// alternatively we could return []byte or even RawBytes
-			return scanTypeString
-		}
-		return scanTypeNullString
+		return scanTypeRawBytes
```

Comment:
Do I have to check nullable to decide converting to sql.NullString or string myself?

Reply:
`sql.RawBytes` can also be `nil`, which is distinguishable from an empty string (`""`). I.e. if `rawBytes == nil` then it was a NULL value.
If you want to known if the column itself is nullable, then you should check `ColumnType.Nullable()`